assignment2/src/com/company/main/Student.java

```diff
@@ -0,0 +1,89 @@
+package com.company.main;
+import java.util.Comparator;
+import java.util.List;
+import java.io.Serializable;
+
+public class Student implements Serializable{
+    public String name, addr, courses;
+    public int age, rollNo;
+
+    public void Assign(String name, String addr,String courses, int age, int rollNo){
+        this.name = name;
+        this.addr = addr;
+        this.courses = courses;
+        this.age = age;
+        this.rollNo = rollNo;
+    }
+    public String getName()
+    {
+        return name;
+    }
+
+    public String getAddr()
+    {
+        return addr;
+    }
+
+    public String getCourses()
+    {
+        return courses;
+    }
+
+    public int getAge()
+    {
+        return age;
+    }
+
+    public int getRollNo()
+    {
+        return rollNo;
+    }
+
+    @Override
+    public String toString(){
+        return (name +"\t\t"+ age +"\t\t"+ addr + "\t\t" + rollNo + "\t\t" + courses +" ");
+    }
+
+    public static Comparator<Student> stuNameComparator = new Comparator<Student>() {
```

Comment:
static methods already have context of the calling class, does the prefix stu add more context ??